### PR TITLE
fix: added missing 'ORDER BY' before 'OFFSET' II

### PR DESCRIPTION
**Pull request by frbuceta**
If the supportsOffsetFetch is enabled, the generated SQL query is invalid as it does not comply with the expected syntax of a SELECT - ORDER BY Clause.

This PR resolves this issue by prepending the missing ORDER BY to OFFSET.

_This PR was originally created by @achrinza_

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-mssql) 👈

- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [x] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: pretest.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ if (!process.env.MSSQL_DATABASE) process.env.MSSQL_DATABASE = global.getConfig()
 const catFileCmd = (isWin ? 'type ' : 'cat ') + sqlFileDir;
 
 const sqlcmd = catFileCmd + ' | ' + sqlDependencyDir + ' -s ' + process.env.MSSQL_HOST + ' -o ' + process.env.MSSQL_PORT +
-' -u ' + process.env.MSSQL_USER + ' -p ' + process.env.MSSQL_PASSWORD + ' -d ' + process.env.MSSQL_DATABASE;
+' -u ' + process.env.MSSQL_USER + ' -p "' + process.env.MSSQL_PASSWORD + '" -d ' + process.env.MSSQL_DATABASE;
```
Comment (achrinza) [inline pretest.js]: Thanks for contributing more fixes! Though maybe we should put this as a separate PR to keep things clean?

Comment (frbuceta) [reply pretest.js]: Ready

**Comment by frbuceta** [comment]
ready for merge
